packages/react/src/ActionList/Item.tsx

```diff
@@ -294,10 +295,11 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
       'data-inactive': inactive ? true : undefined,
       'data-loading': loading && !inactive ? true : undefined,
       tabIndex: disabled || showInactiveIndicator ? undefined : 0,
-      'aria-labelledby': `${labelId} ${slots.inlineDescription ? inlineDescriptionId : ''}`,
-      'aria-describedby': slots.blockDescription
-        ? [blockDescriptionId, inactiveWarningId].join(' ')
-        : inactiveWarningId,
+      'aria-labelledby': `${labelId} ${slots.trailingVisual ? trailingVisualId : ''}`,
+      'aria-describedby':
```

Comment:
Untested code, that reads slightly better:

```suggestion
      'aria-describedby': [
        slots.blockDescription ? blockDescriptionId : undefined,
        inactiveWarningId ?? undefined,
        slots.inlineDescription ? inlineDescriptionId : undefined,
      ]
        .filter(String)
        .join(' '),
```

Reply:
Looks like it worked! Thanks for the suggestion!